keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ClusterActiveServerGroup.kt

```diff
@@ -0,0 +1,53 @@
+package com.netflix.spinnaker.keel.clouddriver.model
```

Comment:
should scope this stuff under ec2 (or aws probably, using cloudProvider as the key) as the model will be totally different on another cloud provider

Reply:
Good point